knowledge-base/grid-edit-on-row-double-click.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Edit a row on click or double-click with a popup form
+description: How to edit a row on click or double-click with a popup form
+type: how-to
+page_title: Edit a row on click or double-click with a popup form
+slug: grid-edit-row-click
+position: 
+tags: 
+ticketid: 1443681
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I would like to edit a row in the Grid when the user clicks or double-clicks on it, instead of using the command buttons.
+
+## Solution
+
+The Grid exposes two events that allows you to respond to the user clicking on its rows - [OnRowClick]({%slug grid-events%}#onrowclick) and [OnRowDoubleClick]({%slug grid-events%}#onrowdoubleclick). You can use either one of them together with the [Window]({%slug components/window/overview%}) to create a custom popup form on a click of a row. 
+
+>caption Use the OnRowDoubleClick event to open a custom popup form to edit the Grid
+
+````CSHTML
+@* Click on a Grid row twice to see the custom edit form *@ 
+
+<TelerikGrid Data="@MyData"
+             Height="400px"
+             Width="700px"
+             Pageable="true"
+             OnRowDoubleClick="@OnRowDoubleClickHandler">
+    <GridColumns>
+        <GridColumn Field="@(nameof(SampleData.Id))" Width="120px" />
+        <GridColumn Field="@(nameof(SampleData.Name))" Title="Employee Name" Groupable="false" />
```

Comment:
We can remove the Groupable = "false" as this Grid is not groupable. I suppose it is left from the Grid overview example if you got it from there.